test/plots/barPlotTests.ts

```diff
@@ -773,6 +773,20 @@ describe("Plots", () => {
         assert.include(["visible", "inherit"], label2.style("visibility"), "label 1 is visible");
         svg.remove();
       });
+
+      it("hides labels that are partially cut off in y", () => {
+        yScale.domain([1, 2]);
+        let texts = svg.selectAll("text");
+
+        assert.strictEqual(texts.size(), 2, "There should be two labels rendered");
+
+        let label1 = d3.select(texts[0][0]);
+        let label2 = d3.select(texts[0][1]);
+
+        assert.strictEqual(label1.style("visibility"), "hidden", "label 1 is not visible");
+        assert.strictEqual(label2.style("visibility"), "hidden", "label 2 is not visible");
```

Comment:
Can we have more information on why these labels are invisible?  From what I gather the labels are not visible because the y scale places them outside the top and bottom of the area.  Can we instead use a `forEach` loop, check through the DOM attributes, and then assert the visibility style based on the DOM attributes?